homeassistant/components/elgato/button.py

```diff
@@ -40,14 +40,14 @@ class ElgatoButtonEntityDescription(
 BUTTONS = [
     ElgatoButtonEntityDescription(
         key="identify",
-        name="Identify",
+        translation_key="identify",
         icon="mdi:help",
         entity_category=EntityCategory.CONFIG,
         press_fn=lambda client: client.identify(),
     ),
     ElgatoButtonEntityDescription(
         key="restart",
-        name="Restart",
+        translation_key="restart",
```

Comment:
We should add translations for device classes which can be reused in integrations. Let's do that in separate PRs, and make this PR draft until that's been done.

Reply:
I've now re-used the translations available in [6784252](https://github.com/home-assistant/core/pull/89629/commits/67842521e4884b60b1f32d563dd7ce37769438bc)

I'm not 100% sure this is what you meant.